STL_Extension/doc/STL_Extension/Concepts/Hashable.h

```diff
@@ -6,10 +6,10 @@
 A type `Key` is a model of the concept `Hashable` if the
 specializations `boost::hash<Key>` and `std::hash<Key>` exist.
 
-\cgalHasModel All handles and indices of \cgal data structures.
-\cgalHasModel All handles of OpenMesh, by including the specializations
-of the `graph_traits` header files provided by \cgal.
-They can be disables by defining the macro `CGAL_DISABLE_HASH_OPENMESH`.
+\cgalHasModelsBegin
+\cgalHasModelsBare{All handles and indices of \cgal data structures}
+\cgalHasModelsBare{All handles of OpenMesh\, by including the specializations of the `graph_traits` header files provided by \cgal. They can be disables by defining the macro `CGAL_DISABLE_HASH_OPENMESH`.}
```

Comment:
another construction not obvious for the dev (meaning we should add it to the dev doc at least)

Reply:
See comment https://github.com/CGAL/cgal/pull/7576#discussion_r1270394580